OpenRA.Game/Server/Server.cs

```diff
@@ -75,7 +75,7 @@ public sealed class Server
 
 		ReplayRecorder recorder;
 		GameInformation gameInfo;
-		readonly List<GameInformation.Player> worldPlayers = new List<GameInformation.Player>();
+		public readonly List<GameInformation.Player> worldPlayers = new List<GameInformation.Player>();
```

Comment:
```suggestion
		public readonly List<GameInformation.Player> WorldPlayers = new List<GameInformation.Player>();
```

Reply:
This is because of change of scope right

Reply:
Yes